### PR TITLE
react-bootstrap, modal, tooltip, reusable card

### PR DESCRIPTION
**Pull request by hihaianh**
- Made and styled a reusable card component (free for everyone to use and may need tweaking to cater for events page)
- Modal: Added pop-up for ‘create’ button on broadcast component
- Tootltip: Able to provide information when hovering over icon next to message (inside modal)
- React-bootstrap dependency installed
- Bootstrap (global) styles imported in main.jsx; without it, modal won’t appear in the centre and I'll change button colour on broadcast component by overriding it with an id/another color later on

### CODE REVIEW

**Review by cactusballs — COMMENTED** [review]
This looks amazing, Well done Anh, sickening work friend! 

A couple of my comments are more questions, but might be differences in bootstrap, feel free to disagree with anything :)

File: client/src/App.css [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #root {
-  max-width: 1280px;
+  /* commenting out styles thats affecting the image on login/homepage */
+  /* max-width: 1280px; */
```
[the remark this answers is not in the document]
Comment (hihaianh) [reply client/src/App.css:3]: We can just remove it- doesn't seem to be affecting anything else but the login image.

File: client/src/components/dashboard/Broadcast.css [hunk 2]
```diff
@@ -12,49 +13,78 @@
...
-  padding: 10px 0px;
+  padding: 0px 0px;
+  height: 78px;
   background-color: #485f39;
   border-radius: 10px;
   color: white;
   text-align: center;
-  font-size: 24px;
+  font-size: 22px;
   font-family: "League Spartan";
```
[the remark this answers is not in the document]
Comment (hihaianh) [reply client/src/components/dashboard/Broadcast.css:23]: Yup we can give that a go.

File: client/src/components/dashboard/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          {linkText}
+        </a>
+      </div>
+      {/* card content- will display default message when there's no 'content', else it'll will render anything under content */}
+      <div className="card-content">
+        {content || "No content at the moment"}
+      </div>
+      {/* footer is optional (when there's no button at the bottom of card) e.g., event cards */}
+      {/* footer is only rendered when you supply a button prop */}
+      {(footer || footerButton) && (
```
[the remark this answers is not in the document]
Comment (hihaianh) [reply client/src/components/dashboard/Card.jsx]: I can rename it to cardFooter.

**Pull request merged**